frescobaldi_app/hyphdicts/hyph_de_DE.dic

```diff
@@ -17122,10 +17122,10 @@ fuchs1pelz
 fuchs1rot
 fug
 fugato
-fuge
+fugue
 f�ge
```

Comment:
It seems there are some non-utf8 characters. Can this be fixed as well?

Reply:
In that directory some files are ASCII or ISO-8859; only the English file is UTF-8.

```
$ file *.dic
hyph_cs_CZ.dic: troff or preprocessor input, ISO-8859 text
hyph_da_DK.dic: ISO-8859 text
hyph_de_DE.dic: ISO-8859 text
hyph_el_GR.dic: ISO-8859 text
hyph_en_CA.dic: ASCII text
hyph_en_GB.dic: ASCII text
hyph_en_US.dic: Unicode text, UTF-8 text
hyph_es_ES.dic: ISO-8859 text
hyph_fi_FI.dic: ISO-8859 text
hyph_fr.dic:    troff or preprocessor input, ISO-8859 text
hyph_ga_IE.dic: ISO-8859 text
hyph_hu.dic:    troff or preprocessor input, ISO-8859 text
hyph_id_ID.dic: ASCII text
hyph_is_IS.dic: ISO-8859 text, with CRLF line terminators
hyph_it.dic:    troff or preprocessor input, ASCII text
hyph_nl_NL.dic: troff or preprocessor input, ISO-8859 text
hyph_nn_NO.dic: troff or preprocessor input, ISO-8859 text
hyph_pl.dic:    troff or preprocessor input, ISO-8859 text
hyph_pt_BR.dic: ISO-8859 text
hyph_pt_PT.dic: ISO-8859 text
hyph_ru_RU.dic: ISO-8859 text
hyph_sk_SK.dic: ISO-8859 text
hyph_sv_SE.dic: ASCII text
hyph_uk_UA.dic: ISO-8859 text
```

I think we should leave it as it is now, see:

- https://www.chromium.org/developers/how-tos/editing-the-spell-checking-dictionaries/
- https://cgit.freedesktop.org/libreoffice/dictionaries/plain/de/de_DE_frami.dic

Reply:
Ok, in that case I'll give an approve!